datahub-web-react/src/app/settings/AccessTokenModal.tsx

```diff
@@ -60,7 +60,7 @@ export const AccessTokenModal = ({ visible, onClose, accessToken, expiresInText
             onCancel={onClose}
             footer={
                 <>
-                    <Button id="createTokenButton" onClick={onClose}>
+                    <Button id="createTokenButton" onClick={onClose} data-testid="close-button">
```

Comment:
can we name this access-token-modal-close-button?

usually we like to have data-testid to be more specific

Reply:
sure